scripts/dts/extract_dts_includes.py

```diff
@@ -349,8 +349,13 @@ def extract_property(node_compat, yaml, node_address, prop, prop_val, names,
             # Generate alias definition if parent has any alias
             if parent_address in aliases:
                 for i in aliases[parent_address]:
-                    node_alias = i + '_' + def_label
-                    aliases[node_address].append(node_alias)
+                    # Build an alias name that respects device tree specs
```

Comment:
Can you explain this further, I see this change the alias defines from:

I2C_1_NXP_FXOS8700_1F_BASE_ADDRESS 

to 

I2C_1_FXOS8700_1F_BASE_ADDRESS

Trying to understand what the issue was.

Reply:
node_alias build was of form: i2c1_NXP_FXOS8700_1F (or close to that),
and this named as injected in edts
It should rather be build using  i2c1 + "node_name" to build an close to real alias, and then converted to a define label style, so value put in edts looks like a real alias